tests/templates/test_subroutines/test_select.py

```diff
@@ -206,6 +207,14 @@ def test_flatten_unflatten(self):
         assert op.target_wires == new_op.target_wires
         assert op is not new_op
 
+    def test_copy(self):
+        """Test that the copy function of Select works correctly."""
+        ops = [qml.PauliX(wires=2), qml.PauliX(wires=3), qml.PauliY(wires=2), qml.SWAP([2, 3])]
```

Comment:
Can you replace one of these with a parameterized op like an RZ or something. Copying the data should ensure that we transfer the parameters of each sub op into our copied op. This doesn't test that as is.